.github/workflows/build_and_test.yml

```diff
@@ -530,6 +530,10 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/workflows/windows-tester-prepare
+      # Ensure we're using nightly, because libafl-libfuzzer requries it
+      - uses: dtolnay/rust-toolchain@nightly
```

Comment:
it requires?

Reply:
Yeah, we need it for linker shenanigans in the build script sadly